x-pack/plugins/cloud_security_posture/public/plugin.tsx

```diff
@@ -78,9 +78,11 @@ export class CspPlugin
         (
           <KibanaContextProvider services={{ ...core, ...plugins }}>
             <RedirectAppLinks coreStart={core}>
-              <SetupContext.Provider value={{ isCloudEnabled: this.isCloudEnabled }}>
-                <CspRouter {...props} />
-              </SetupContext.Provider>
+              <div style={{ width: '100%', height: '100%' }}>
```

Comment:
at some point if pref ever becomes an issue for us we could start with removing `{{` that break immutability